src/security-council-mgmt/governors/SecurityCouncilNomineeElectionGovernor.sol

```diff
@@ -215,11 +216,16 @@ contract SecurityCouncilNomineeElectionGovernor is
     ///         during the vetting phase and exclude any contenders which dont meet this criteria.
     /// @dev    Can be called only while a proposal is pending (after proposal created but before voting phase)
     ///         A contender cannot be a member of the opposite cohort.
-    function addContender(uint256 proposalId) external {
```

Comment:
maybe let's keep the old addContender too and mark it as deprecated / revert

Reply:
like this? 72c7c399a121ddb69db87635c3aefca6dab19a15